ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestConfig.java

```diff
@@ -151,6 +151,13 @@
   private static final String KSQL_HEARTBEAT_THREAD_POOL_SIZE_CONFIG_DOC =
       "Size of thread pool used for sending / processing heartbeats and cluster discovery.";
 
+  public static final String KSQL_QUERY_STANDBY_ENABLE_CONFIG =
```

Comment:
Not sure if we want this extra nob. It allows for controlling heartbeat and routing to standbys independently. However, do we want heartbeat by itself without routing to standbys?